src/inmanta/moduletool.py

```diff
@@ -489,11 +489,15 @@ def do_update(specs: "Dict[str, List[InmantaModuleRequirement]]", modules: List[
                     allow_pre_releases=my_project.install_mode != InstallMode.release,
                     use_pip_config=my_project.metadata.pip.use_config_file,
                 )
+                # Invalidate ast cache so that dependencies of installed modules can be updated as well
+                my_project.invalidate_state()
```

Comment:
I'm not sure I see how this can be done as the pip install command install all the modules at once. I don't see another solution than saving `project.modules` before and after the installation and looking for changes. Would this be ok?

Reply:
You are right. We should keep it like it was.